src/UID2.Client/KeyContainer.cs

```diff
@@ -104,6 +110,11 @@ public bool TryGetMasterKey(DateTime now, out Key key)
             return TryGetKeysetActiveKey(_masterKeysetId, now, out key);
         }
 
+        public bool IsDomainNameForSite(int siteId, string domainName)
```

Comment:
Note sure about this name, any suggestions?

Reply:
IsDomainNameAllowedForSite ?

Reply:
Renamed as above